src/rpc/core_rpc_server.h

```diff
@@ -285,6 +285,31 @@ namespace cryptonote
       std::cout << "Votes and uptime relayed";
       loki::write_redirected_stdout_to_shared_mem();
     }
+
+    void on_debug_mine_singular_block(std::string const &address)
+    {
+      cryptonote::miner &miner = m_core.get_miner();
+      if (miner.is_mining())
+      {
+        std::cout << "Already mining";
```

Comment:
`\n` here and below? Not sure how you use these though.

Reply:
Yeah should be ok, the intention is to send line by line over to the receiving end and parsed out, so the less formatting codes the easier to deal with.